includes/admin/assets.php

```diff
@@ -86,9 +86,13 @@ public function load()
 			SIMPLE_CALENDAR_VERSION,
 			true
 		);
-		$run_oauth_helper = get_option('simple_calendar_run_oauth_helper');
 
-		if ($run_oauth_helper && $this->current_page === 'simple-calendar_settings') {
+		if (
+			(is_plugin_active('Simple-Calendar-Google-Calendar-Pro-main/simple-calendar-google-calendar-pro.php') ||
```

Comment:
I don't think this is the path when you install by zip. Also, is there a better way to handle this without specifying path?